src/libdredd/src/mutation_replace_unary_operator.cc

```diff
@@ -39,6 +39,31 @@ bool MutationReplaceUnaryOperator::IsPrefix(clang::UnaryOperatorKind op) {
   return op != clang::UO_PostInc && op != clang::UO_PostDec;
 }
 
+std::string MutationReplaceUnaryOperator::GetExpr(
```

Comment:
Please apply analogous changes to this class as per my suggestions for "binary".

Reply:
Done.